src/components/GlobalNav/GlobalNav.tsx

```diff
@@ -23,15 +24,17 @@ export interface NavMenuItem {
 export interface NavProps {
   leftLinks: NavMenuItem[];
   rightLinks: NavMenuItem[];
+  socialLinks: NavMenuItem[];
   currentSite: string;
-  secondaryNavDesktop?: JSX.Element;
-  secondaryNavMobile?: JSX.Element;
+  secondaryNavDesktop: any;
```

Comment:
do we need to type these as `any`? It looks like we are always passing in `JSX.Element`.